test/transforms/projectionpursuit.jl

```diff
@@ -0,0 +1,71 @@
+@testset "ProjectionPursuit" begin
+  N = 100_000
+  a = [2randn(rng, N÷2) .+ 6; randn(rng, N÷2)]
+  b = [3randn(rng, N÷2); 2randn(rng, N÷2)]
+  c = randn(rng, N)
+  d = c .+ 0.6randn(rng, N)
+  t = (;a, b, c, d)
+  
+  T = ProjectionPursuit()
+  n, c = apply(T, t)
+
+  @test Tables.columnnames(n) == (:a, :b, :c, :d)
+  
+  μ = mean(Tables.matrix(n), dims=1)
+  @test isapprox(μ[1], 0; atol=1e-6)
+  @test isapprox(μ[2], 0; atol=1e-6)
+  @test isapprox(μ[3], 0; atol=1e-6)
+  @test isapprox(μ[4], 0; atol=1e-6)
+
+  Σ = cov(Tables.matrix(n))
+  @test isapprox(Σ[1,2], 0; atol=1e-6)
+  @test isapprox(Σ[1,3], 0; atol=1e-6)
+  @test isapprox(Σ[1,4], 0; atol=1e-6)
+  @test isapprox(Σ[2,1], 0; atol=1e-6)
+  @test isapprox(Σ[2,3], 0; atol=1e-6)
+  @test isapprox(Σ[2,4], 0; atol=1e-6)
+  @test isapprox(Σ[3,1], 0; atol=1e-6)
+  @test isapprox(Σ[3,2], 0; atol=1e-6)
+  @test isapprox(Σ[3,4], 0; atol=1e-6)
+  @test isapprox(Σ[4,1], 0; atol=1e-6)
+  @test isapprox(Σ[4,2], 0; atol=1e-6)
+  @test isapprox(Σ[4,3], 0; atol=1e-6)
+  @test isapprox(Σ[1,1], 1; atol=1e-6)
+  @test isapprox(Σ[2,2], 1; atol=1e-6)
+  @test isapprox(Σ[3,3], 1; atol=1e-6)
+  @test isapprox(Σ[4,4], 1; atol=1e-6)
+
+  if visualtests
+    p₁ = corner(t, title="Original")
+    p₂ = corner(n, title="Transformed")
```

Comment:
Please pass the option `size=(..., ...)` to adjust the plots. The current size is inhibiting the visualization of the marginal distributions. Also, notice that the first variable is not Gaussian after the transform so there must be an issue somewhere with the code, or we need more iterations to fix it.

Reply:
Hi @DianaPat did you have a chance to take a look into this final adjustment? We just need to make sure that the two datasets are plotted correctly. We can take care of the remaining adjustments moving forward.

Reply:
Hi @juliohm, I have not worked much of it this last few days, but for what I have checked, the marginal distributions don't plot correctly even if I change the size. The PairPlot options don't work either. I think it's a matter of the gr definition, as this only happen when I set it up.

Reply:
I think you need to increase the plot size a bit more so that the marginals don't get shrunk. It is a Plots.jl issue. Can you try with a larger size to see if it works? Something large like 1000x1000 for example just to see if it works, and then you can decrease to a minimum size that works.

Reply:
It doesn't work.